docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -735,28 +563,86 @@ As database IDs are all globally unique, it's fine to replace table ID with data
 
```

Comment:
Old row nr 724
```
* Unpartitioned index: the concatenation of table id and index id, e.g. `100_1`
```
This seems to imply that we will support global indexes, (but not with ranges?)
Maybe remove this line?

Old row nr 731
```
As database IDs are all globally unique, it's fine to replace table ID with database ID in the key range. For example, assuming the database ID is 100, then the string format of its key range is:
```
Are they unique in the set of table_id's as well? Or should we simply add a new prefix for database ids? like 'd_<database_id>' or rather 's_<schema_id>'?

Reply:
> Are they unique in the set of table_id's as well?

Yes, the current implentation assumed that for simplicity. But adding other prefixes is fine. It is an implementation detail.